reana/reana_benchmark/utils.py

```diff
@@ -0,0 +1,29 @@
+#
+# This file is part of REANA.
+# Copyright (C) 2021 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Contains common functions shared by reana-benchmark modules."""
+
+import logging
+from datetime import datetime
+
+from reana.reana_benchmark.config import DATETIME_FORMAT
+
+logging.basicConfig(
```

Comment:
I need `logger` otherwise a lot of unnecessary logs are coming from `reana-client`. Agree, `utils` is not the best place.  I can move `logging.basicConfig` to `cli.py` but I cannot move `logger` there because it will create a circular dependency (other modules like `submit`, etc. need a `logger`)

Some possible solutions:

1. Leave only `logger` in `utils.py`, move `logging.basic` to `cli`

2. Init `logger` in each file separately, move `logging.basic` to `cli`

3. Put `logging.basic` and `logger` to a `config.py`

4. Have a dedicated file for logging like `logging.py` or similar

I am inclined towards solutions `1` or `3`.

Reply:
After some thinking, solution `1` seems a better option. Having basic config in `cli.py` (entry point) makes more sense.

Reply:
Implemented solution `1`. What do you think?

Reply:
Looks good